src/render/blog/business-logic-definition.html.md

```diff
@@ -0,0 +1,99 @@
+---
+title: What does business logic mean for us
+description: We will find an answer what does the business logic mean for us. We would transform user stories to domain objects and then model the solution.
+author: tomaszmichalak
+keywords: business,requirements,user-stories,business-logic,domain-object,domain-space,solution-space
+order: 1
+date: 2020-06-04
+---
+
+This is the first article in the series dedicated to the subject of business logic.
```

Comment:
Let's remove that, we will add this info when more articles in the series will come.